mLRS/Common/hal/esp/esp-device_conf.h

```diff
@@ -149,6 +149,13 @@
   #define FREQUENCY_BAND_2P4_GHZ
 #endif
 
+#ifdef TX_ELRS_RADIOMASTER_TX16S_2400_ESP32
```

Comment:
I guess I would prefer having an indication that it's an internal in the def name

Reply:
You choose this name in your owpr-esp-tx-internal branch and I changed to match what you chose when I looked at your branch in the middle of getting this going.  grumble grumble

Reply:
me having choosen something in some initial thing long ago isn't a good argument :D:D:D

I guess TX_ELRS_INTERNAL_xxx or TX_ELRS_INT_xxx would give some easy to catch naming scheme which also would do some good ordering